services/blockchain-indexer/shared/dataService/business/network.js

```diff
@@ -13,21 +13,37 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { requestConnector } = require('../utils/request');
+const { requestConnector } = require('../../utils/request');
+const { getAvailableModuleCommands, getRegisteredModules } = require('../../constants');
 
-const business = require('./business');
+const getNetworkStatus = async () => {
+	const status = await requestConnector('getNetworkStatus');
 
-const getPeers = async params => {
+	status.moduleCommands = await getAvailableModuleCommands();
+	status.registeredModules = await getRegisteredModules();
+	status.constants = { chainID: status.chainID };
```

Comment:
Won't `chainID` be present twice now? Once as `status.chainID` and another as `status.constants.chainID`?

Reply:
@priojeetpriyom Yes, may be this line is unnecessary but i would recommend not to remove now. We can check and remove it later after beta.

Reply:
Can we please create or mention in some exiting card just to keep it noted?